sql/src/main/java/io/crate/execution/ddl/tables/AlterTableRequest.java

```diff
@@ -101,30 +99,25 @@ public Settings settings() {
     }
 
     @Nullable
-    private static String source(Map<String, Object> mappingSource) {
-        if (mappingSource.isEmpty()) {
+    private static String serialize(Map<String, Object> mappingDelta) throws IOException {
```

Comment:
what about `mapToJson` to express what is done? also here `mappingDelta` may be missleading, it's just a Map represention. If this is just a delta or not is out of scope.